_docs/setup/kubernetes/multicluster-install.md

```diff
@@ -0,0 +1,178 @@
+---
+title: Istio Multicluster
+overview: Install Istio with multicluster support.
+
+order: 65
+
+layout: docs
+type: markdown
+---
+
+{% include home.html %}
+
+Instructions for the installation of Istio multicluster.
+
+## Prerequisites
+
+* Two or more Kubernetes clusters.
+
+* The usage of an RFC1918 network, VPN, or other more advanced network techniques
+to meet the following requirements:
+
+    * Individual cluster Pod CIDR ranges and service CIDR ranges must be unique
+across the multicluster environment and may not overlap.
+
+    * All nodes' pod CIDR in every cluster must be routable to every other nodes'
+pod CIDR.
+
+    * All Kubernetes control plane API servers must be routable to each other.
+
+* Helm **2.7.2 or newer**.  The use of Tiller is optional.
+
+* If using Tiller, the ability to modify RBAC rules is required to install Helm
+or alternatively Helm should already be installed.
+
+* Kubernetes **1.7.3 or newer**.
+
+* Istio secret `istio.default` replicated to all remote Kubernetes clusters
+from the Istio control plane per application namespace.
+
+* The local cluster and all remote Kubernetes cluster credentials (typically
+stored in `$HOME/.kube/config`) copied to
+`$HOME/multicluster/${UNIQUE_IDENTIFIER}.kube.conf`.
+
+* A [deployed Istio control plane]({{home}}/docs/setup/kubernetes/quick-start.html)
+on **one** Kubernetes cluster.  Istio must be [deployed with Helm]({{home}}/docs/setup/kubernetes/helm-install.html) using the `--set global.multicluster.enabled=true` flag.  This is currently mandatory.
+
+* Currently only [manual sidecar injection]({{home}}/docs/setup/kubernetes/sidecar-injection.html#manual-sidecar-injection)
+has been validated with multicluster.
+
+## Caveats and known problems
+
+<img src="{{home}}/img/exclamation-mark.svg" alt="Warning" title="Warning" style="width: 32px; display:inline" />
+All known caveats and known problems with multicluster for the 0.8 release are [tracked here](https://github.com/istio/istio/issues/4822).
+
+## Install a multicluster configmap on the Istio control plane
+
+<img src="{{home}}/img/exclamation-mark.svg" alt="Important" title="Important" style="width: 32px; display:inline" />
+Pilot will not start until the configmap in these instructions
```

Comment:
This seems to be a short-term limitation - I think the plan (with the code using programmatic loading of configmap) is to watch for the config map to show up and change. But it's probably ok for now, dynamic adding of clusters is low priority

( this is mainly due to primary cluster loaded from configmap - will be fixed in 0.8 with the secrets change)